src/click_select/index.es

```diff
@@ -10,7 +10,28 @@ class ClickSelect extends PureComponent {
     this.handleClick = this.handleClick.bind(this);
   }
 
-  handleClick(event) {
+  copyToClipboard() {
+    if (!document.queryCommandSupported) return false;
```

Comment:
read browser specifics here: https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript

try to use the new Clipboard API first.